database/redis/metric.go

```diff
@@ -128,7 +130,9 @@ func (connector *DbConnector) SaveMetrics(metrics map[string]*moira.MatchedMetri
 				continue
 			}
 
-			if err = c.Publish(ctx, metricEventKey, event).Err(); err != nil {
+			rand.Seed(time.Now().UnixNano())
```

Comment:
Usually getting time is an expensive operation. I suggest doing it one time.

Reply:
> I suggest doing it one time.

How exactly do you propose to do this?

Reply:
I suggest moving the line `rand.Seed(time.Now().UnixNano())` before line `for _, metric := range metrics {`.

Reply:
Done